src/test/scala/dotty/semanticdb/Tests.scala

```diff
@@ -111,8 +111,8 @@ class Tests {
     val scalaFolderReg = """scala-(\d+)\.(\d+)""".r
     val (_, _, path) = files
       .collect(file =>
-        file.getName match {
-          case scalaFolderReg(major, minor) => (major, minor, file.getName)
+        file.nn.getName match {
```

Comment:
A Java method/field that would need a more precise type annotation than just null/non-null at the outermost level.